test-plugin/src/main/java/io/papermc/paper/testplugin/PreTestPlugin.java

```diff
@@ -0,0 +1,16 @@
+package io.papermc.paper.testplugin;
+
+import io.papermc.paper.plugin.Initializer;
+import io.papermc.paper.registry.RegistryAccess;
+import org.bukkit.Bukkit;
+
+public class PreTestPlugin implements Initializer {
+    @Override
+    public void initialize() {
+        // TODO: Loggers? We could expose the provider logger? Idk a nice way tho
```

Comment:
Ideally we must load the logger before the plugin load for the initializer (only if there are some initializer)

Reply:
I think plugin can create their own logger and paper can provide an utility method to create a logger with the plugin prefix without doing that manually, cause take a logger in the parameter sounds a little bit weird